app/lib/discount.rb

```diff
@@ -0,0 +1,30 @@
+module Discounts
+
+  def discount_total
+    discount = 0
+    discount += discount_total_price
+    discount += discount_muffins
+    discount.round(2)
+  end
+
+  def discount_total_price
+    discount_level = 5
+    discount_applied_at = 50
+    net_total > discount_applied_at ?
+      calculate_discount(net_total, discount_level) : 0
+  end
+
+  def discount_muffins
+    discount_level = 10
+    matched_order_lines = order.select { |line| line[:item].include?('Muffin') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.